src/connector/src/parser/unified/debezium.rs

```diff
@@ -37,6 +38,44 @@ pub const DEBEZIUM_DELETE_OP: &str = "d";
 pub const DEBEZIUM_TRANSACTION_STATUS_BEGIN: &str = "BEGIN";
 pub const DEBEZIUM_TRANSACTION_STATUS_COMMIT: &str = "END";
 
+pub fn parse_transaction_meta(
+    accessor: &impl Access,
+    connector_props: &ConnectorProperties,
+) -> std::result::Result<TransactionControl, AccessError> {
+    if let (Some(ScalarImpl::Utf8(status)), Some(ScalarImpl::Utf8(id))) = (
+        accessor.access(&[TRANSACTION_STATUS], Some(&DataType::Varchar))?,
+        accessor.access(&[TRANSACTION_ID], Some(&DataType::Varchar))?,
+    ) {
+        // The id field has different meanings for different databases:
+        // PG: txID:LSN
+        // MySQL: source_id:transaction_id (e.g. 3E11FA47-71CA-11E1-9E33-C80AA9429562:23)
+        match status.as_ref() {
+            DEBEZIUM_TRANSACTION_STATUS_BEGIN => match *connector_props {
+                ConnectorProperties::PostgresCdc(_) => {
+                    let (tx_id, _) = id.split_once(':').unwrap();
+                    return Ok(TransactionControl::Begin { id: tx_id.into() });
+                }
+                ConnectorProperties::MysqlCdc(_) => return Ok(TransactionControl::Begin { id }),
```

Comment:
Where do we depend on the value of the ID?

Reply:
Currently we have a check in https://github.com/risingwavelabs/risingwave/pull/14375/files#diff-d20a701c6d209a35e84545b11ccd13ab2f332a68aa175eb4bf73edf67184e8bcR673-R675, if transaction id mismatch, it will print a warn log.

Reply:
Oh, I see. IIUC, the LSN for `BEGIN` and `COMMIT` differs from each other?

Reply:
Yes, lsn in commit will greater than begin.